### ISSUE TITLE
HelpBrowser websocket connection fail

### ISSUE THREAD

**Issue by patrickdupuis** [supercollider/supercollider#4256]
Environment
-----------

* Your SuperCollider version: 3.10.1-rc1 [bb7773cb0]
* Your operating system and version: Fedora 28
* Qt version: 5.11.1

Steps to reproduce (for bugs)
-----------------------------

Open `sclang` in a terminal and evaluate `HelpBrowser.instance`

Error message (for bugs)
------------------------

```
QObject::connect: No such slot QtCollider::WebView::onRenderProcessTerminated(RenderProcessTerminationStatus, int)
-> a HelpBrowser
sc3> SCDoc: Indexing help-files...
SCDoc: Indexed 1988 documents in 1.8 seconds
JavaScript Message: WebSocket connection to 'ws://localhost:12344/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED
JavaScript Message: WebChannel error: [object Event]
```

Expected Behavior
-----------------

The HelpBrowser should open without posting any errors.

Current Behavior
----------------

The HelpBrowser opens, causing the above error messages, but appears to function properly.

**Comment by patrickdupuis** [comment]
On Ubuntu 18.04 with SC 3.10.1-rc1 [0754d8b47], I get the same error message as above. Interestingly, If I close the Help browser window and try creating a new one, I end up with a blank unresponsive window and am forced to quit sclang.

**Comment by patrickdupuis** [comment]
I'm still able to reproduce this issue on latest `3.10`.

**Comment by snappizz** [comment]
sorry about the delay, #4267 will fix this

**Comment by patrickdupuis** [comment]
#4267 does indeed fix this on Fedora 30 at least. I still have to test on Ubuntu.

**Comment by patrickdupuis** [comment]
On Ubuntu, I no longer see the websocket connection error message. I am able to open multiple instances of the IDE also.
There's still something causing `HelpBrowser.instance` to be unresponsive in command line sclang. I think that this is a separate issue and won't be fixed by #4267.

**Comment by brianlheim** [comment]
IIUC, this is now completely fixed.
